src/sys/time.rs

```diff
@@ -118,6 +116,7 @@ pub(crate) mod timer {
                         libc::timespec {
                             tv_sec: 0,
                             tv_nsec: 0,
+                            ..
```

Comment:
What's the function of this line?  Are you programming defensively, just in case freebsd, netbsd, dragonfly, or illumos grow some padding fields too?

Reply:
It's required once Rust starts targeting musl 1.2 as `i686-unknown-linux-musl` will have a padding field:

```
$ cargo +e392e7b50913d5cb6e2c08d70cdeb72e56d289f9 build --target i686-unknown-linux-musl
   Compiling libc v0.2.137 (https://github.com/wesleywiser/libc?branch=musl-1.2#3049e59a)
   Compiling nix v0.25.0 (/tmp/nix)
error: pattern requires `..` due to inaccessible fields
   --> src/sys/time.rs:116:25
    |
116 | /                         libc::timespec {
117 | |                             tv_sec: 0,
118 | |                             tv_nsec: 0,
119 | |                         },
    | |_________________________^
    |
help: ignore the inaccessible and unused fields
    |
118 |                             tv_nsec: 0, ..,
    |                                       ++++
```

Reply:
Oh, my bad.  I saw that `#[cfg()]` block on line 100 and thought that it applied here.

Reply:
No problem, thanks for the quick reviews! 🙂